### DISCUSSION TITLE
Set max solar sell value

### DISCUSSION THREAD

**Discussion by leleweb1979** [kellerza/sunsynk#283] [Q&A]
Hi, it is possible to set the max solar sell value?

**Comment by Hatios** [comment]
Specifically max solar sell, or max export?

I do not believe there is a way to limit specifically solar export, but if your device is set to sell excess solar, and not export otherwise, the general max export setting can be used to achieve this.

**Comment by kellerza** [comment]
You could try these, I have never tested them before so please give some feedback if this works for you

https://github.com/kellerza/sunsynk/blob/main/src/sunsynk/definitions/single_phase.py#L229

**Comment by Hatios** [comment]
I can confirm that Max Sell Power does work as I use this to regulate my export.
Please note this is not solar specific and regulates the max export from any source on the inverter.
The Max Solar Power also does work, but that limits the power the inverter will draw from the panels. You are almost certainly going to want that left as high as it can go as it is related to all generation, not just export.
